R/uvhydrograph-styles.R

```diff
@@ -1,11 +1,19 @@
 
 getUvStyle <- function(data, info, correctionLabels, plotName) {
-  x <- data[[1]]$time
-  y <- data[[1]]$value
+  if (exists('value', where=data[[1]])) { #point data
+    x <- data[[1]]$time
+    y <- data[[1]]$value
+  } else { #rectangle
```

Comment:
I don't think you need any of this - you can just stick data[[1]]$x0 into the list directly rather than making an entirely new object. For the rectangles, the x and y values that already exist (x <- data[[1]]$time) will just be NULL. Which should never cause an error because the rectangle calls don't even use x and y